### PR TITLE
add header component

### PR DESCRIPTION
**Pull request by Ali7040**
# Related Issues: Add header component #146 

## Description
I added a header component and also changed the favicon.

## What type of PR is this? (check all applicable)
- [x] 🅰️ Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Mobile & Desktop Screenshots/Recordings
[Attach screenshots or recordings if applicable]

## Added to documentation?
- [ ] 📜 README.md
- [ ] 🙅 no documentation needed

## [Optional] Post-deployment tasks
[Specify any post-deployment tasks that need to be performed]

## [Optional] What gif best describes this PR or how it makes you feel? 
[Embed gif or describe the feeling in plain text]

### CODE REVIEW

File: angular-v17-AnguMAT/src/app/components/header/header.component.html [hunk 1]
```diff
@@ -1,14 +1,62 @@
 <mat-toolbar>
-  <span>My App</span>
+  <mat-toolbar-row>
+    <a routerLink="/" class="logo">
+      <img
+        src="../../../assets/images/angular-headless-hashnode-logo.jpg"
```
Comment (yshashi) [inline angular-v17-AnguMAT/src/app/components/header/header.component.html:5]: How about using an absolute path `/assets/images/angular-headless-hashnode-logo.jpg`

Comment (Ali7040) [reply angular-v17-AnguMAT/src/app/components/header/header.component.html:5]: "Yeah, but we did the same thing in the `angular-v17 `folder, so I prefer that next time I try to make it dynamic using property binding.

> How about using an absolute path `/assets/images/angular-headless-hashnode-logo.jpg`

**Review by yshashi — APPROVED** [review]
Good job man!

File: angular-v17-AnguMAT/src/app/components/header/header.component.html (hunk 1 above)
Comment (monacodelisa) [reply angular-v17-AnguMAT/src/app/components/header/header.component.html:5]: @Ali7040  exactly - the intention is for the blog image to eventually be connected to the blog image of the blog owner, this is just a temp placeholder image for the meantime

**Review by monacodelisa — APPROVED** [review]

**Comment by monacodelisa** [comment]
@Ali7040 thank you 👍

**Pull request merged**